packages/formation/sass/base/_b-typography.scss

```diff
@@ -0,0 +1,238 @@
+html {
```

Comment:
This file was [copied from `uswds`](https://github.com/uswds/uswds/blob/v1.6.14/src/stylesheets/elements/_typography.scss), then the `va-back-to-top` changes were made.